lld/MachO/BPSectionOrderer.cpp

```diff
@@ -60,6 +61,30 @@ getRelocHash(const Reloc &reloc,
   return getRelocHash(kind, sectionIdx.value_or(0), 0, reloc.addend);
 }
 
+// Get a hash of the unwind info (after relocation).
+// This hash is not 100% accurate, but it's good enough for compression.
```

Comment:
nit: s/accurate/unique? (Or representative?) 
(I'm not sure saying a hash string being "accurate" makes sense )

On the similar topic, it's possible to have more than one personality symbol *with the same name*, with one being from the dynamic library (eg., libc++) and the other being a user-defined one. (We've seen this in the past use cases).
Should their symbol type be included in the hash?

Reply:
One key point is we want these hashes to be deterministic. Otherwise builds could end up with non-determinstic function order, which is what the `lld/test/MachO/bp-section-orderer-stress.s` aims to prevent. So this is why we cannot simply hash the personality pointer, even though that is what we'd like to do. I think it makes sense to include the symbol type in the hash if that's possible.

Reply:
Proposing `Hash(lsda->getName() + personality->getName() + utohexstr(encoding))` (name is "<null>" if it is a nullpointer)

Reply:
> nit: s/accurate/unique? (Or representative?)

I think I'll just remove that line of comment to avoid confusion.